doc/client.rst

```diff
@@ -362,6 +362,9 @@ Batch Operations
 
         Perform the same read/write transactions on multiple keys.
 
+        .. note:: There is a bug in Python client < 14.0.0 where only passing read operations does not work. This bug
```

Comment:
I'm not sure what this sentence "only passing read operations" means. Can you clarify that?

Reply:
It means passing in only read operations to the `ops` parameter in `batch_operate()`. Read operations don't change any data in the server, it only reads it.

Reply:
OK, how about we edit this line to read: "Prior to Python client 14.0.0, using the `batch_operate() method with only `read` operations caused an error. This bug was fixed in version 14.0.0."

Reply:
That's fine. I verified that's the behavior for Python client versions < 14.0.0